File: kyw0716/level0/옹알이(1).js
function solution(babbling) {
  let answer = 0;

  for (let i = 0; i < babbling.length; i++) {
    if (babbling[i].replace(/aya|ye|woo|ma/g, "").length === 0) answer++;
  }

  return answer;
}

// https://school.programmers.co.kr/learn/courses/30/lessons/120956

File: sk000801/입문/옹알이(1).java
class Solution {
    public int solution(String[] babbling) {
        String[] baleum = {"aya", "ye", "woo", "ma"};
        int answer = 0;
        for (String s : babbling) {
            String origin = s;
            for (String ss : baleum) {
                if (s.contains(ss)) {
                    s = s.replace(ss, " ");
                }
            }
            s = s.replaceAll(" ", "");
            if (s.equals("")) {
                answer++;
            }
        }
        return answer;
    }
}